packages/eslint-plugin-react-hooks/src/index.js

```diff
@@ -9,18 +9,45 @@
 
 import RulesOfHooks from './RulesOfHooks';
 import ExhaustiveDeps from './ExhaustiveDeps';
+import {name, version} from '../package.json';
```

Comment:
Can we `require` this instead to avoid having to use the rollup plugin? 

Alternatively, you could also hardcode the name and use `version` from `~/Reactversions.js`. But let's try `require` first.

Reply:
Updated